packages/datadog-instrumentations/src/jest.js

```diff
@@ -176,21 +176,21 @@ function cliWrapper (cli) {
     const configurationPromise = new Promise((resolve) => {
       onDone = resolve
     })
-    if (!jestConfigurationCh.hasSubscribers) {
+    if (!jestItrConfigurationCh.hasSubscribers) {
       return runCLI.apply(this, arguments)
     }
 
     sessionAsyncResource.runInAsyncScope(() => {
-      jestConfigurationCh.publish({ onDone })
+      jestItrConfigurationCh.publish({ onDone })
     })
 
     try {
-      const { err, config } = await configurationPromise
+      const { err, itrConfig } = await configurationPromise
       if (err) {
         log.error(err)
       }
-      isCodeCoverageEnabled = config.isCodeCoverageEnabled
-      isSuitesSkippingEnabled = config.isSuitesSkippingEnabled
+      isCodeCoverageEnabled = itrConfig.isCodeCoverageEnabled
```

Comment:
when `err` happens, has `itrConfig` also value? If it hasn't, we will have an exception here.

Reply:
It's caught but yeah, you're right. I'll make sure `itrConfig` always has a value